packages/react-server-redux/package.json

```diff
@@ -0,0 +1,29 @@
+{
+  "name": "react-server-redux",
+  "version": "0.5.1",
+  "description": "Package to support Redux in React-Server",
```

Comment:
I think it might be self-evident its a package, so maybe just "Redux support for React Server"?